### ISSUE TITLE
free batchnormalization layer for resnet50

### ISSUE THREAD

**Issue by smilesun** [marrlab/DomainLab#211]
According to https://arxiv.org/pdf/2007.01434.pdf
appendix D, 

"
For the architecture “Resnet-50”, we replace the final (softmax) layer of a ResNet50 pretrained on
ImageNet and fine-tune. Observing that batch normalization interferes with domain generalization
algorithms (as different minibatches follow different distributions), we freeze all batch normalization
layers before fine-tuning. We insert a dropout layer before the final linear layer.
"

**Comment by Car-la-F** [comment]
2. freeze bathnorm: 
can be done using 
```
net_torchvision.<name of a batchnormalisation layer>.requires_grad_(False)
```

From short investigation I think the batchnormalisation layer are all called `bn1`, `bn2`, ...
One needs to also go into the deeper structures by using net_torchvision.layer1, net_torchvision.layer2, ...
